lib/bridge.js

```diff
@@ -94,32 +155,27 @@ Bridge.prototype.list = function list () {
  *
  * @memberOf Bridge#
  * @param {Number}    nukiId              the id of the nuki
- * @param {Boolean}   [doNotValidate]     if true, will not validate existance (by calling the list endpoint of the bridge)
  * @returns {Promise<Nuki>}
  */
-Bridge.prototype.get = function get (nukiId, doNotValidate) {
+Bridge.prototype.get = function get (nukiId) {
```

Comment:
Is there a good reason to delete the parameter? It might break existing implementations and does not break anything if left, right?

Reply:
`doNotValidate` is not possible anymore, because the list of Nuki's is required due to the necessity of the `deviceType` for instantiation of the new Nuki.

Reply:
I think this is acceptable

Reply:
Please remove the "test" for the parameter

Reply:
What do you mean by removing the `test` parameter?

Reply:
I mean the test, which calls this function with the parameter and does nothing else. Please remove it